src/View/Widget/MultiCheckboxWidget.php

```diff
@@ -152,6 +153,9 @@ protected function _renderInputs($data, $context)
             if (!isset($checkbox['templateVars'])) {
                 $checkbox['templateVars'] = $data['templateVars'];
             }
+            if (!isset($checkbox['label'])) {
```

Comment:
I was sticking with the same sanity check as `$checkbox['templateVars']` it stops the setting of $checkbox each loop of the foreach. Personally think its a bit pointless, I don't think its saving many cycles ;-)